pr-lines-test.py

```diff
@@ -1,8 +1,5 @@
+import xml
```

Comment:
'The Python documentation recommends using `defusedxml` instead of `xml` because the native Python `xml` library is vulnerable to XML External Entity (XXE) attacks. These attacks can leak confidential data and "XML bombs" can cause denial of service. ([Line 1](https://github.com/chmccreery/test-dev-server/blob/b93064ef819d19258cb35a77faab9807461214eb/pr-lines-test.py#L1))\n#\n```suggestion\nimport defusedxml\n```\n<a href=\'https://dev.semgrep.dev/api/agent/deployments/37/issues/18995/sentiment?hash=af9e24d4f606f15aadcf98dda486865dbb917f4a965f36709905a10f6889f0e5&sentiment=bad&utm_campaign=finding_notification&utm_medium=review_comment&utm_source=github&utm_content=logo\'>\n<img align=\'right\' src=\'https://img.shields.io/badge/-Not%20helpful%20%F0%9F%91%8E-%235F36D9\' height=24px alt=\'\' />\n</a>\n<sub>⚪️ This finding does not block your pull request.\n</sub><br/><sub>\n🙈 From <a href=\'https://semgrep.dev/r/python.lang.security.use-defused-xml.use-defused-xml?utm_campaign=finding_notification&utm_medium=review_comment&utm_source=github&utm_content=rule\'>python.lang.security.use-defused-xml.use-defused-xml</a>.\n</sub>\n\n<!--\n\n🤫 WELCOME TO SECRET SEMGREP! 🤫\nThis information is for debugging purposes and does not appear in rendered PR comments.\n\nFinding id: 18995\nSyntactic id: 3b1709949d372c16a6e8cf5c799b60da\nStart line: 1,1\nEnd line: 1,11\nIndex: 0\n-->'